backend/app/controllers/api/v1/sessions_controller.rb

```diff
@@ -25,7 +25,14 @@ def sign_in
         end
       end
 
-      def sign_out; end
+      def sign_out
+        case log_out
+        in Success
+          render json: nil, status: :ok
```

Comment:
is render nil is good practice?
@npupko

Reply:
`head :ok`